src/main/java/com/example/bookstore/dto/cart/item/CartItemDto.java

```diff
@@ -0,0 +1,14 @@
+package com.example.bookstore.dto.cart.item;
+
+import lombok.Data;
+
+@Data
+public class CartItemDto {
+    private Long id;
+
+    private Long shoppingCartId;
```

Comment:
Optional: You're returning the value, that user already knows (types in URL path). To my mind, it makes your code a little complicated, and you can just use CartItemDtoWithoutShoppingCartId for returning value always.